cli/src/test/browse.spec.ts

```diff
@@ -20,7 +20,8 @@ test('browse lists templates on dumb terminal', async (t) => {
       description: 'something',
       similarTemplatesIds: [],
       tags: [],
-    }, {
+    },
```

Comment:
Why?  To my eye this just wastes space, and violates the spirit of this holy dictum https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace:
> The basic principle is: The more code that fits on one screen, the easier it is to follow and understand the control flow of the program. Use whitespace purposefully to provide separation in that flow.

I prefer to nestle commas here, if you don't have a strong preference otherwise.  If this linter configuration _requires_ it, I will argue here strongly for not requiring it.  I would prefer to force the nestled version here.

Reply:
didn't try to start a war. simply enforcing a little more with minimal changes. removed elements alignment.